src/test/java/easytests/core/models/IssueModelTest.java

```diff
@@ -1,50 +1,36 @@
 package easytests.core.models;
 
 import easytests.core.entities.IssueEntity;
-import easytests.core.models.empty.ModelsListEmpty;
-import easytests.core.models.empty.SubjectModelEmpty;
-import org.junit.Assert;
+import easytests.support.IssueSupport;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.meanbean.test.*;
-import org.mockito.Mockito;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.meanbean.test.ConfigurationBuilder;
+
 
 /**
- * @author fortyways
+ * @author greenbarrow
  */
-@RunWith(SpringRunner.class)
-@SpringBootTest
-public class IssueModelTest {
+public class IssueModelTest extends AbstractModelTest {
+
+    private IssueSupport issueSupport = new IssueSupport();
+
+    @Override
+    protected ConfigurationBuilder getConfigurationBuilder() {
+        return super.getConfigurationBuilder()
+                .ignoreProperty("subject");
```

Comment:
remove ignore, add support meanbean subject

Reply:
got it